### DISCUSSION TITLE
Recover Prometheus data (not persisted)

### DISCUSSION THREAD

**Discussion by brospars** [prometheus-operator/prometheus-operator#4731] [Q&A] [answered]
Hello,

Is there a way to recover prometheus pod data from a restarted pod (following a config patch) ?

I was running `prometheus-operator` without a PV for some time by mistake. I've updated the config but lost all the data...

**Comment by simonpasquier** [comment, marked as answer]
If you haven't configured persistent storage, the answer is no unfortunately. Unless maybe you have access to the node and the container data hasn't been erased yet.

**Reply by brospars** [reply]
Ok thanks that what I thought. My bad 😅
